lib/transfer.cpp

```diff
@@ -254,7 +254,7 @@ namespace quda {
     ColorSpinorField &coarse(*coarse_tmp_h);
 
     // compute the coarse grid point for every site (assuming parity ordering currently)
-    for (int i=0; i<fine.Volume(); i++) {
+    for (unsigned int i = 0; i < fine.Volume(); i++) {
```

Comment:
since volume is `size_t`  this should also be `size_t`.

Reply:
Not necessarily.  The volume here is always representable as a 32 bit value.  The move to making it 64 bit was to ensure that expressions involving the volume are computed correctly.

Reply:
a0c68c0